src/login/loginlog.cpp

```diff
@@ -13,14 +13,14 @@
 #include <common/sql.hpp>
 #include <common/strlib.hpp>
 
-// local sql settings
-static char   log_db_hostname[64] = ""; // Doubled to reflect the change on commit #0f2dd7f
-static uint16 log_db_port = 0;
-static char   log_db_username[32] = "";
-static char   log_db_password[32] = "";
-static char   log_db_database[32] = "";
-static char   log_codepage[32] = "";
-static char   log_login_db[256] = "loginlog";
+
+int log_db_port = 3306;
+std::string log_db_ip = "127.0.0.1";
+std::string log_db_id = "ragnarok";
+std::string log_db_pw = "";
+std::string log_db_db = "ragnarok";
+std::string log_login_db = "loginlog";
+std::string log_codepage = "";
```

Comment:
Please keep the variable names and dont rename them.
And if you could keep the same order it would also be nice.